### DISCUSSION TITLE
Any news on a release

### DISCUSSION THREAD

**Discussion by lordratner** [toverainc/willow-inference-server#150] [General]
Hey there,

Just checking in, it's been quiet as I'm sure life has stepped in, but is the WIS still being worked on?

Cheers!

**Comment by ceinstaller** [comment]
I'm starting to wonder this myself.  Hope everything is OK with everyone involved.

**Comment by nikito** [comment]
I think the devs have just been busy with their day-jobs and life, there was some mention about things in the discord. We still have active discussions on discord,  and there are some changes being made on wis, they just haven't been finalized yet.

**Reply by ceinstaller** [reply]
Great news!
